addons/bus/static/tests/bus_connection_alert.test.js

```diff
@@ -31,6 +31,7 @@ test("show warning when bus connection encounters issues", async () => {
         WEBSOCKET_CLOSE_CODES.ABNORMAL_CLOSURE
     );
     await waitForSteps(["reconnecting"]);
+    await runAllTimers();
     const alert = await waitFor(".o-bus-ConnectionAlert");
     expect(alert).toHaveText("Real-time connection lost...");
```

Comment:
Would be best as `await advanceTime(CONNECTION_LOST_WARNING_DELAY);` rathher than `runAllTimers()`.

This clarifies the UI is displayed after 15 seconds, consuming all `setTimeout()` in stack is too technical and obscure.

_This test is the main one: it introduces the UI feedback on connection lost. The 15 seconds are important.
The other test is about reconnection attempt, which is more specific. It's also good to see exact timing there too._